src/os/style/style.js

```diff
@@ -302,7 +356,8 @@ os.style.DEFAULT_VECTOR_CONFIG = {
   // this will only be applied to line and polygon types
   'stroke': {
     'width': os.style.DEFAULT_STROKE_WIDTH,
-    'color': os.style.DEFAULT_LAYER_COLOR
+    'color': os.style.DEFAULT_LAYER_COLOR,
+    'lineDash': os.style.DEFAULT_LINE_STYLE
```

Comment:
This property should not be included at all if the line is not dashed.

Reply:
Does this go for the highlight and select styles too?
I noticed that cesium doesn't show any select or highlight style without the line dash specified in those configs

Reply:
Also - removing this seems to get the state test to pass.

Reply:
Yes, it should be undefined by default for all styles. That sounds like a bug introduced in these changes, given we don't currently use `lineDash` anywhere.